### PR TITLE
bump oasis-core 20.10

### PR DESCRIPTION
**Pull request by ptrus**
Fixes: #1006

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -20,8 +20,9 @@ oasis-ethwasi-runtime-common = { path = "./common" }
 oasis-ethwasi-runtime-keymanager = { path = "./keymanager-runtime" }
 
 # Oasis core.
-oasis-core-runtime = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.9" }
-oasis-core-keymanager-client = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.9" }
+oasis-core-runtime = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.10" }
+oasis-core-keymanager-client = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.10" }
+ed25519-dalek = { git = "https://github.com/dalek-cryptography/ed25519-dalek", tag = "1.0.0-pre.3" }
```
Comment (kostko) [inline Cargo.toml:25]: Why is this needed and wasn't needed before?

Comment (ptrus) [reply Cargo.toml]: it's not needed, just needed to be updated in cargo.lock (and this triggered that on build :-)), reverted the change

File: Cargo.toml [hunk 2]
```diff
@@ -40,11 +40,12 @@ io-context = "0.2.0"
 serde_bytes = "0.11"
 serde_json = "1.0"
 sha3 = "0.6"
+log = "0.4"
```
Comment (nhynes) [inline Cargo.toml:43]: please sort deps alphabetically

File: common/Cargo.toml [hunk 3]
```diff
@@ -5,8 +5,8 @@ authors = ["Oasis Labs Inc. <info@oasislabs.com>"]
 
 [dependencies]
 # Oasis Core.
-oasis-core-runtime = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.9" }
-oasis-core-keymanager-client = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.9" }
+oasis-core-runtime = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.10" }
+oasis-core-keymanager-client = { git = "https://github.com/oasisprotocol/oasis-core", tag = "v20.10" }
```
Comment (nhynes) [inline common/Cargo.toml:9]: sort

File: gateway/Cargo.toml [hunk 4]
```diff
@@ -5,7 +5,7 @@ license = "GPL-3.0"
 authors = ["Oasis Labs Inc. <info@oasislabs.com>", "Parity Technologies <admin@parity.io>"]
 
 [dependencies]
-log = "0.3"
+log = "0.4"
```
Comment (nhynes) [inline gateway/Cargo.toml:8]: please sort (I know they're not already)

**Review by nhynes — APPROVED** [review]
